apps/Learning-Buddy/screens/SignUpScreen.jsx

```diff
@@ -97,7 +97,9 @@ export const SignUpScreen = ({ navigation }) => {
           />
         </View>
         {/* This button will need to pass values to auth process in future iterations */}
```

Comment:
Would you be so kind as to remove this comment before you pull? This is no longer required. :)

Reply:
It is done